pyproject.toml

```diff
@@ -30,16 +30,17 @@ packages = [
 "Documentation" = "https://pandas.pydata.org/pandas-docs/stable"
 
 [tool.poetry.dependencies]
-python = ">=3.9"
+python = ">=3.9 <3.13"
```

Comment:
I assume that is needed because one of the dependencies added this constraint? Might be nicer to move this constraint to that specific dependency. Something like `{ version = ">=x.y.z" , python = "<4"} `

xref #625

Reply:
It now pulls in numpy 1.26.0 and scipy 1.11.2, which have the python < 3.13 restriction, so I put that in.